dispatchr-ruby/app/controllers/requests_controller.rb

```diff
@@ -1,53 +1,66 @@
 class RequestsController < ApplicationController
+	before_action :set_request, only: [:show, :update, :destroy]
 
-
-	#GET
+	#GET /requests
 	def index
 		render json: Request.all
 	end
 
-	#GET
+	#GET /requests/1
 	def show
-		request = Request.find(params[:id])
-		render json: request
+		render json: @request
+	end
+
+	#GET /requests/user/?user_id=1
+	def search_user
+		#get user_id from URL
+		url = request.original_url
+		uri = URI.parse(url)
+		params = CGI.parse(uri.query)
+
+		#find all Requests where user_id matches
+		arr = []
+		Request.where(user_id: "#{params['user_id'].first}").find_each do |req|
```

Comment:
```ruby
@requsts = Request.where(user_id: params[:id])
```
This returns what you can think of as an array or requests.
Then you can do ``` render json: @requests``` to return the list of requests in JSON format